indra/newview/llfloateremojipicker.cpp

```diff
@@ -27,10 +27,18 @@
 
 #include "llfloateremojipicker.h"
 
+#include "llcombobox.h"
+#include "llemojidictionary.h"
 #include "llfloaterreg.h"
+#include "lllineeditor.h"
 #include "llscrolllistctrl.h"
 #include "llscrolllistitem.h"
-#include "llemojidictionary.h"
+#include "lltextbox.h" 
+#include "llviewerchat.h" 
+
+std::string LLFloaterEmojiPicker::mSelectedCategory;
```

Comment:
Should these be declared elsewhere?

Reply:
> Should these be declared elsewhere?

These are instances of static variables, they are usually defined at the top of cpp module

Reply:
In such case it's name is wrong. It should have been sSelectedCategory;

Reply:
I usually declare them as static in the header/class definition.  Andrey is correct though - we usually prefix statics with an 's' to distinguish them from non-static class members.